integration_test/curl-dcm.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# url, output_file
+curl -X GET \
+     -H "Authorization: Bearer "$(gcloud auth print-access-token) \
+     $1 \
+     |tail -c+137 | head -c-68 >$2
```

Comment:
Are you using head and tail here to strip the multipart boundaries? You can instead specify the accept header application/dicom; transfer-syntax=* to get the instance without a boundary.

Reply:
Yeah, tail and part solution is ugly, changed to header. Weird though that stow-rs standard says that response is always multipart, even for single instance.